cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -653,7 +607,7 @@ When all the packages and dependencies have been installed, and the configuratio
 parameters set, you can start the services on each of the CDAP boxes by running the
 command::
 
-  $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i restart ; done
+  $ for i in /etc/init.d/cdap* ; do sudo service $i restart ; done
```

Comment:
this wont work... it would need to be ``sudo $i restart`` since  ``service`` wont recognize ``/etc/init.d/cdap-master`` as a service name.  i think the original is better, to preserve the use of ``service``